src/pages/Tools/AddressConversion/MultiVersionAddress.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { MultiVersionAddress as MultiVersionAddressType } from './types'
+import CopyableText from '../../../components/CopyableText'
+
+export const MultiVersionAddress: React.FC<{
+  multiVersionAddr: MultiVersionAddressType
+  displayName?: boolean
+}> = ({ multiVersionAddr, displayName }) => {
+  return (
+    <>
+      {displayName && multiVersionAddr.name && (
+        <div>
+          <strong>Lock:</strong> {multiVersionAddr.name}
+        </div>
+      )}
+      <div>
+        <strong>CKB2021:</strong> <CopyableText>{multiVersionAddr.ckb2021FullFormat}</CopyableText>
+      </div>
+      {multiVersionAddr.ckb2019ShortFormat && (
+        <div>
+          <strong>CKB2019(short format):</strong> <CopyableText>{multiVersionAddr.ckb2019ShortFormat}</CopyableText>
+        </div>
+      )}
```

Comment:
The deprecated address can be removed, it's been long time since the new one has been adopted, so keeping the deprecated one is misleading.